core/modules/filter/filter.pages.inc

```diff
@@ -156,6 +159,29 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
     '#parents' => array('attributes', 'height'),
     '#field_suffix' => ' ' . t('pixels')
   );
+  $available_style_options = array();
```

Comment:
```suggestion
  $available_style_options = array(
    NULL => t('Original image'),
  );
```

This lets people stick with, or reset to, the original image. Additionally, the field can't be set to `#required`. Or maybe there's a different UI that makes more sense. Perhaps a checkbox "Reset image" which would hide the image styles dropdown and would also reset the width/height (which still appear on the `<img>` tag even if the width/height fields don't appear in the modal).

Reply:
Some interesting thoughts... :thinking: 
I'm leaning towards "the admin decides whether orig image is allowed or not" (of course, it should be possible to enable orig). But if it's allowed, toggling should be possible, of course.

Reply:
After some restructure, resetting to original is now possible only be switching the dropdown value.